src/Plugins/ApplicationLogs/Store/States/EngineLogState.cs

```diff
@@ -49,11 +49,12 @@ public virtual void Serialize(BinaryWriter writer)
 
         #region IEquatable
 
-        public bool Equals(EngineLogState other) =>
+        public bool Equals(EngineLogState? other) =>
+            other is not null &&
```

Comment:
Really 🤣 _have to be different_
```suggestion
            other != null &&
```